setup.py

```diff
@@ -76,7 +76,7 @@ def run_tests(self):
         "requests>=2.0,<3",
         "pycryptodome==3.9.6",
         "python-magic>=0.4,<1",
-        "Jinja2>=2.0,<3",
+        "Jinja2>=3.0",
```

Comment:
we should bound it by <4 in case the next major version breaks this

Reply:
Good point